linera-core/src/client.rs

```diff
@@ -2508,54 +2482,61 @@ where
             .await
     }
 
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
What was the reason for that again? We will need to have this functionality available in Wasm, too.

Reply:
`zstd` compression doesn't work on wasm, unfortunately :/

Reply:
But before this change, `publish_bytecode` was unconditionally compiled. And it contained `contract.into()`—isn't that what compresses the bytecode?

Anyway, if this doesn't work, we should have a `TODO` and an issue to track this: We do probably want to have this available in Wasm, e.g. for browser plugins.